.github/CODEOWNERS

```diff
@@ -0,0 +1,194 @@
+/tutorials/editor/*script*                          @godotengine/documentation @godotengine/script-editor
+
+# TODO: Who should control this beyond docs?
+
+## --------------- Export --------------------------
+
+/tutorials/export/*android*                         @godotengine/documentation @godotengine/android
+/tutorials/export/*ios*                             @godotengine/documentation @godotengine/ios
+/tutorials/export/*linux*                           @godotengine/documentation @godotengine/linux-bsd
+/tutorials/export/*macos*                           @godotengine/documentation @godotengine/macos
+/tutorials/export/*visionos*                        @godotengine/documentation @godotengine/ios
+/tutorials/export/*web*                             @godotengine/documentation @godotengine/web
+/tutorials/export/*windows*                         @godotengine/documentation @godotengine/windows
+
+# TODO
+
+## --------------- i18n ----------------------------
+
+# TODO: Who should control this beyond documentation?
+
+## --------------- Inputs --------------------------
+
+/tutorials/inputs/                                  @godotengine/documentation @godotengine/input
+
+## --------------- IO ------------------------------
+
+# TODO: Who should control this beyond documentation?
+
+## --------------- Math ----------------------------
+
+# TODO: Who should control this beyond documentation?
+
+## --------------- Migrating -----------------------
+
+/tutorials/migrating/                               @godotengine/documentation
+
+## --------------- Navigation ----------------------
+
+/tutorials/navigation/                              @godotengine/documentation @godotengine/navigation
+
+## --------------- Networking ----------------------
+
+/tutorials/networking/                              @godotengine/documentation @godotengine/network
+
+## --------------- Performance ---------------------
+
+# TODO: Who should control this beyond documentation?
+
+## --------------- Physics -------------------------
+
+/tutorials/physics/                                 @godotengine/documentation @godotengine/physics
+
+## --------------- Platform ------------------------
+
+/tutorials/platform/android/                        @godotengine/documentation @godotengine/android
+/tutorials/platform/ios/                            @godotengine/documentation @godotengine/ios
+/tutorials/platform/web/                            @godotengine/documentation @godotengine/web
+
+# TODO
+
+## --------------- Plugins -------------------------
+
+# TODO: Who should control this beyond documentation?
+
+## --------------- Rendering -----------------------
+
+/tutorials/rendering/                               @godotengine/documentation @godotengine/rendering
+
+## --------------- Scripting -----------------------
+
+# TODO: Who should control this in general beyond documentation?
```

Comment:
Scripting is technically owned by Core, but the gdscript and dotnet teams should be better familiar with the APIs. The information in here seems to be a mix of all, maybe we need to do it on a case by case basis.